app/templates/menu-search.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>{{ title }}</title>
+        <style>
+
+            .code {
+                background-color: #d8d1de;
+                color: #45474a;
+            }
+
+            .form {
+				width: 50%;
+                padding-top: 20px;
+				padding-bottom: 50px;
+				padding-left: 33%;
+				float: center;
+                border: 4px solid #873ebb;
+                border-radius: 10px;
+				background-color: #fbf2ff;
+				margin-left: 10%;
+				margin-top: 70px;
+				font-weight: bold;
+            }
+			
+			.center {
+				margin-left: 0%;
+			}
+
+        </style>
+    </head>
+    <body class="code">
+        <h1>{{ title }}</h1>
+
+        <form action="" class="form">
```

Comment:
action should make a request to the endpoint:
i.e
{{ url_for('routes.menu-browse') }}

it should also specify the request type to be POST